### PR TITLE
Clarify shape_centered property

### PR DESCRIPTION
**Pull request by groud**
Following #33440

### CODE REVIEW

File: doc/classes/TouchScreenButton.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
 			The button's shape.
 		</member>
 		<member name="shape_centered" type="bool" setter="set_shape_centered" getter="is_shape_centered" default="true">
-			If [code]true[/code], the button's shape is centered.
+			If [code]true[/code], the button's shape is centered in the provided texture. If no texture are used, this property has no effect.
```
Comment (akien-mga) [inline doc/classes/TouchScreenButton.xml]: *is* used

Comment (groud) [reply doc/classes/TouchScreenButton.xml]: Done

**Comment by akien-mga** [comment]
Thanks!

**Comment by follower** [comment]
One comment from the future:

AFAICT "centered *in* the provided texture" is only true when the shape is smaller than the texture.

If the shape is larger than the provided texture (which seems to be a common use case to allow for "sloppy" interactions on touch devices) then it's the *texture* which is centered in the shape.

If this is the intended behaviour, I wonder if "centered *on* the provided texture" might be more accurate?

**Pull request merged**